<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>559</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Data Mining App</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f7f7f7;
color: #000000;
font-family: Grotesco;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #242424</string>
   </property>
   <widget class="QScrollArea" name="scrollPictureArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #5c5c5c</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>539</width>
       <height>399</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="gridFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #292929</string>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
     <item row="1" column="1">
      <widget class="QPushButton" name="playButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
     background-color:rgba(255,255,255,30);
     border: 1px solid rgba(255,255,255,40);
     border-radius:7px;
width: 100;
height: 30;
}
QPushButton:hover{
background-color:rgba(255,255,255,50);
}
QPushButton:pressed{
background-color:rgba(255,255,255,70);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="LabelPictureArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-1</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Grotesco</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Исходное изображение/видеопоток</string>
    </property>
   </widget>
   <widget class="QFrame" name="rightGridFrame">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>211</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #292929</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Открыть</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionOpenVideo"/>
    <addaction name="actionOpenCamera"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenImage">
   <property name="text">
    <string>Изображение</string>
   </property>
  </action>
  <action name="actionOpenVideo">
   <property name="text">
    <string>Видеопоток</string>
   </property>
  </action>
  <action name="actionOpenCamera">
   <property name="text">
    <string>Веб-камера</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
